<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>btnBrowseOutputFolder</class>
 <widget class="QWidget" name="btnBrowseOutputFolder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="cmbClassificationCriteria">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>By year</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>By year and month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>By year, month and day</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkUseCopy">
        <property name="text">
         <string>Copy instead of moving files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkAddFoldersForExtensions">
        <property name="text">
         <string>Add folders for file extensions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSearchRecursively">
        <property name="text">
         <string>Search recursively</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Source folder:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="btnRun">
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output folder:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtOutputFolder"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnBrowseSourceFolder">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Classify by:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtSourceFolder"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="txtLog">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
